### DISCUSSION TITLE
有些图片的行列号的识别精度较差

### DISCUSSION THREAD

**Discussion by jweden33** [RapidAI/TableStructureRec#104] [Q&A]
有些图片的行列号的识别精度较差，如何微调来提升模型效果呀

**Comment by Joker1212** [comment]
具体是哪个模型呢？如果是有线的话，因为是别人贡献，可能你需要自己训练一个unet的模型，如果是无线模型，可以参考致谢里面的阿里读光的仓库进行微调

**Reply by jweden33** [reply]
是读光的那个有线模型，怎么训练一个unet模型呢 请大佬指点一下

**Reply by Joker1212** [reply]
我们的wiredV2不是读光的模型，比它的精度高很多，也是默认有线模型，训练的话，可以参考 https://github.com/jiangnanboy/table_structure_recognition
https://github.com/chineseocr/table-detect
